### PR TITLE
Hotfix: Setting Default ProviderTimeout for api and function

### PR DESCRIPTION
**Pull request by raghits**
### Requirements

* Provide providerTimeout: 160 default for api and function

### Description of the Change

* Sets the default providerTimeout to 160 seconds for API and Function

### Benefits

Doesn't break the build

### Possible Drawbacks

None

### Applicable Issues

None

### CODE REVIEW

File: core/jazz_create-serverless-service/index.js [hunk 1]
```diff
@@ -351,6 +351,19 @@ var getServiceData = (service_creation_data, authToken, configData, deploymentTa
     if (service_creation_data.service_type === "api" || service_creation_data.service_type === "function" || service_creation_data.service_type === "sls-app") {
       serviceMetadataObj.providerRuntime = service_creation_data.runtime;
     }
+    
+    //Adding providerTimeout key in service catalog
+    if (service_creation_data.service_type === "api" || service_creation_data.service_type === "function") {
+      if (service_creation_data.timeout)
```
Comment (devsatishm) [inline core/jazz_create-serverless-service/index.js]: User won't provide the timeout value during service creation. We can skip this check.

File: core/jazz_create-serverless-service/index.js [hunk 2]
```diff
@@ -351,6 +351,19 @@ var getServiceData = (service_creation_data, authToken, configData, deploymentTa
...
+      {
+        serviceMetadataObj.providerTimeout = service_creation_data.timeout;
+      }
+      else
+      {
+        // default to 160 seconds
+        serviceMetadataObj.providerTimeout = 160;
```
Comment (devsatishm) [inline core/jazz_create-serverless-service/index.js]: Move this to configuration. Also, 30 sec is a good start (API Gateway max time out is 29 sec).

Comment (raghits) [reply core/jazz_create-serverless-service/index.js]: Updated.

File: core/jazz_create-serverless-service/index.js (hunk 1 above)
Comment (raghits) [reply core/jazz_create-serverless-service/index.js]: Updated.

**Comment by devsatishm** [comment]
Changes are tested in a trial environment, looks good!

**Pull request merged**